extensions/amp-apester-media/0.1/monetization/companion/video.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Services} from '../../../../../src/services';
+
+/**
+ * @param {!JsonObject} media
+ * @param {AmpElement} apesterElement
+ * @param {!JsonObject} consentObj
+ * @return {Promise}
+ */
+export function handleCompanionVideo(media, apesterElement, consentObj) {
+  const monetizationSettings = media['campaignData'] || {};
+  const companionCampaignOptions =
+    monetizationSettings['companionCampaignOptions'] || {};
+  const companionRawSettings = monetizationSettings['companionOptions'] || {};
+  if (
+    companionRawSettings.video &&
+    companionRawSettings.video.enabled &&
+    companionRawSettings.video.provider === 'sr'
+  ) {
+    const position = getCompanionPosition(companionRawSettings.video);
+    if (position && position !== 'floating') {
+      const macros = getSrMacros(
+        media,
+        companionCampaignOptions['companionCampaignId'],
+        apesterElement,
+        consentObj
+      );
+
+      constructCompanionSrElement(
+        companionRawSettings['video']['videoTag'],
+        position,
+        /** @type {!JsonObject} */ (macros),
+        apesterElement
+      );
+    }
+  }
+  return Promise.resolve();
+}
+/**
+ * @param {!JsonObject} video
+ * @return {string}
+ */
+function getCompanionPosition(video) {
+  if (video['companion']['enabled']) {
+    return 'above';
+  } else if (video['companion_below']['enabled']) {
```

Comment:
Since you are returning early in these checks, `else` here is extraneous and makes the intent of the code less clear.